frontend/src/Contexts/Tasks.tsx

```diff
@@ -9,46 +10,53 @@ import { getAllTasks } from "../services";
 
 export interface TaskInterface {
   id: number;
-  deliveryTime: Date;
+  date: Date;
   isCompleted: boolean;
+  volunteer: any;
   deliveries: MealDeliveryInterface[];
-  name: string; // I will just assume the recipient's name is part of the Task entity for now.
 }
 
 export interface MealDeliveryInterface {
   id: number;
-  quantity: number;
+  isCompleted: boolean;
+  routePosition: number;
   mealType: string;
-  task: TaskInterface;
+  program: any;
```

Comment:
could we make enum types for mealType and program like in the backend? would make it more robust :)

Reply:
updated!